test/testMetagutil.pl

```diff
@@ -38,9 +37,8 @@
 my $gff_scrt = 'gff_file.gff';
 my $prodigal_cmd = '/kb/runtime/bin/prodigal';
 
-my $rast_impl = new RAST_SDK::RAST_SDKImpl();
 
-my $ctx = LocalCallContext->new($token, $auth_token->user_id);
+my $rast_impl = new RAST_SDK::RAST_SDKImpl();
```

Comment:
It is much safer to initialise perl classes using the syntax `Module::Name->new( ... )`.